aries-site/src/examples/components/button/ButtonSizingExample.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Box, Button } from 'grommet';
+
+export const ButtonSizingExample = () => {
+  return (
+    <Box direction="row" align="center" pad="large" gap="medium">
+      <Button label="Default" />
+      <Button size="small" label="Small" />
+      <Button size="medium" label="Medium" />
```

Comment:
I'm wondering if there's a better way to demonstrate that "medium" is the default size. Maybe by ordering them as
`small` `default` `large` ? The display made me wonder "is default the same as medium ..? I think so but I can't tell"